mh/constants.py

```diff
@@ -248,6 +249,27 @@ def slot(item, qty):
 TIME_STATE = 0
 IS_JAP = False
 
+ANNOUNCE = [
+            "<BODY><CENTER>---------- MH3SP ----------<BR>",
+            "<BODY>Monster Hunter 3 (Tri) Server Project",
+            "<BR><LEFT>Welcome to the MH3SP Beta Test! We ask that you",
+            "keep few things in mind while connected in order to",
+            "make the experience as fun as possible for everyone.",
+            "No abusing other players, no exploiting bugs, and",
+            "please report bugs that you find to the server",
+            "staff/dev team. We thank you for your patience as",
+            "we prepared this server to be fit for public use,",
+            "and ask your continued patience in the weeks to",
+            "come as you may encounter some bugs and errors",
+            "while playing. Please see the Public Test Server",
+            "Roadmap for a list of features we are working on.",
+            "<CENTER>Welcome to Loc Lac!<END>"
+]
+if IS_JAP:
+    ANNOUNCE = ANNOUNCE[1:]
+    ANNOUNCE[0] = "<CENTER>" + ANNOUNCE[0]
+ANNOUNCE = "<BR><BODY>".join(ANNOUNCE)
+
```

Comment:
```python
ANNOUNCE = b"<BR><BODY>".join([
    b"<BR><CENTER><BODY>Monster Hunter 3 (Tri) Server Project",
    b"<BR><LEFT>Welcome to the MH3SP Beta Test! We ask that you",
    b"keep few things in mind while connected in order to",
    b"make the experience as fun as possible for everyone.",
    b"No abusing other players, no exploiting bugs, and",
    b"please report bugs that you find to the server",
    b"staff/dev team. We thank you for your patience as",
    b"we prepared this server to be fit for public use,",
    b"and ask your continued patience in the weeks to",
    b"come as you may encounter some bugs and errors",
    b"while playing. Please see the Public Test Server",
    b"Roadmap for a list of features we are working on.",
    b"<BR><CENTER>Welcome to Loc Lac!<END>"
])
```

Reply:
This would remove the logic that adds/removes the first line depending on whether or not it's in japanese mode.

Reply:
That's intended as this line doesn't provide much value and it does work for all versions.

Reply:
Here is a new suggestion using styles as suggested by @InusualZ 
```python
ANNOUNCE = b"<BR><BODY>".join([
    b"<BR><CENTER><BODY><SIZE=6>Monster Hunter 3 (Tri) Server Project",
    b"<BR><LEFT><SIZE=4>Welcome to the MH3SP Beta Test! We ask that you",
    b"keep few things in mind while connected in order to",
    b"make the experience as fun as possible for everyone.",
    b"No abusing other players, no exploiting bugs, and",
    b"please report bugs that you find to the server",
    b"staff/dev team. We thank you for your patience as",
    b"we prepared this server to be fit for public use,",
    b"and ask your continued patience in the weeks to",
    b"come as you may encounter some bugs and errors",
    b"while playing. Please see the Public Test Server",
    b"Roadmap for a list of features we are working on.",
    b"<BR><CENTER><C=2>Welcome to Loc Lac!<END>"
])
```
![image](https://user-images.githubusercontent.com/7890055/210182574-e730e4a5-413d-4495-ae4d-0e6320ae498c.png)

Some notes regarding the tag format:
 - it seems *(I haven't checked the code yet)* the game only picks the first digit after the equal sign *(i.e. `COLOR=1` is the same as `COLOR=10`)*
 - the `<C>` tag seems to be an alias to the `<COLOR>` tag